Sources/Moya/Combine/AnyPublisher+Response.swift

```diff
@@ -0,0 +1,120 @@
+#if canImport(Combine)
+
+import Foundation
+import Combine
+
+#if canImport(UIKit)
+    import UIKit.UIImage
+#elseif canImport(AppKit)
+    import AppKit.NSImage
+#endif
+
+/// Extension for processing raw NSData generated by network access.
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+extension AnyPublisher where Output == Response, Failure == MoyaError {
```

Comment:
You could make it a public extension and avoid have make every method public, like the one in `Sources/Moya/Combine/MoyaProvider+Combine.swift` :))

Reply:
ah good one! I copied the base for it from Rx/Reactive extensions and they have the same issue, gonna fix them all!

Reply:
There's a Swiftlint rule for this if you'd like :]

Reply:
ah nice good idea, added the rule 👍